### PR TITLE
feat: Press Enter key can trigger the Login button

### LINKED ISSUE #1311: feat(fe): Press Enter key can trigger the Login button
# Feature request

## Please describe your feature

At login page, users have to explicitly click the `Login` button to do the login. It's common that also support for triggering login via pressing the `Enter` key. I believe this would be very convenient for those users who always use keyboard instead of mouse, like me. 😄 

## Describe the solution you'd like

After filling in the username and password fields, pressing the Enter key is able to make users do login.

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
resolve #1311 
___
### New feature or improvement
- Describe the details and related test reports.
You can press Enter to log in.
___

### CODE REVIEW

File: web/src/pages/User/Login.tsx [hunk 1]
```diff
@@ -73,6 +73,12 @@ const Page: React.FC = () => {
     });
   };
 
+  const onKeyDown = (e: any) => {
```
Comment (juzhiyuan) [inline web/src/pages/User/Login.tsx:76]: Please use the correct TS typing.

```suggestion
  const onKeyDown = (e: React.KeyboardEvent) => {
```

**Review by juzhiyuan — COMMENTED** [review]
And add the test case when the Enter key is pressed.

**Review by juzhiyuan — APPROVED** [review]

File: web/cypress/integration/user/login.spec.js [hunk 2]
```diff
@@ -44,4 +44,11 @@ context('Login Test', () => {
     cy.contains('Login').click();
     cy.get('.ant-notification-notice-message').should('contain', 'Successfully');
   });
+
+  it('Press Enter to login success', () => {
```
Comment (juzhiyuan) [inline web/cypress/integration/user/login.spec.js:48]:
```suggestion
  it('should press Enter to login successfully', () => {
```

File: web/cypress/integration/user/login.spec.js [hunk 3]
```diff
@@ -44,4 +44,11 @@ context('Login Test', () => {
...
+    cy.visit('/user/Login');
+    cy.get('#control-ref_username').type('user');
```
Comment (juzhiyuan) [inline web/cypress/integration/user/login.spec.js:50]: dom selector

File: web/cypress/integration/user/login.spec.js [hunk 4]
```diff
@@ -44,4 +44,11 @@ context('Login Test', () => {
...
+    cy.get('#control-ref_password').type('user{enter}');
```
Comment (juzhiyuan) [inline web/cypress/integration/user/login.spec.js:51]: `user{enter}`?

Comment (guoqqqi) [reply web/cypress/integration/user/login.spec.js:51]: Yes, there are examples in the official documentation.
image

Comment (juzhiyuan) [reply web/cypress/integration/user/login.spec.js]: what's `enter`?

File: web/cypress/fixtures/data.json [hunk 5]
```diff
@@ -27,5 +27,7 @@
   "updateSuccessfully": "Update Configuration Successfully",
   "deleteSSLSuccess": "Remove target SSL successfully",
   "sslErrorAlert": "key and cert don't match",
-  "pluginErrorAlert": "Invalid plugin data"
+  "pluginErrorAlert": "Invalid plugin data",
+  "successfully": "Successfully",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:31]: No need to take `Successfully` out

File: web/cypress/fixtures/data.json [hunk 6]
```diff
@@ -27,5 +27,7 @@
...
+  "user": "user"
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:32]: ditto

File: web/cypress/integration/user/login.spec.js (hunk 4 above)
Comment (guoqqqi) [reply web/cypress/integration/user/login.spec.js]: image
Information can be found here https://docs.cypress.io/api/commands/type.html#Events

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Hi, please sync your codes with the master branch to fix the CI failure.

**Pull request merged**